Android/app/src/main/java/live/dittolive/chat/SplashActivity.kt

```diff
@@ -55,13 +56,18 @@ class SplashActivity : AppCompatActivity() {
   private fun setupDitto() {
     val androidDependencies = DefaultAndroidDittoDependencies(this)
     DittoLogger.minimumLogLevel = DittoLogLevel.DEBUG
-    DittoHandler.ditto = Ditto(
+    dittoAuthCallback = AuthCallback()
+    ditto = Ditto(
       androidDependencies,
-      DittoIdentity.OfflinePlayground(androidDependencies, BuildConfig.DITTO_APP_ID)
+      DittoIdentity.OnlineWithAuthentication(
+        androidDependencies,
+        BuildConfig.DITTO_APP_ID,
+        dittoAuthCallback,
+        true
```

Comment:
Sure, can you explain what this setting is for? I just assumed it was needed. Maybe we need to add it to the docs or I missed it

Reply:
There is a section on the documentation under transport configuration and there should be inline api documentation, did you find it?

Reply:
The identity service is separate from the synchronization service. So you can identify with a big peer, without syncing to it. does that make sense?

It's a confusing API, and we are working on improving it by allowing you to control what data will sync to the big peer

Reply:
We can discuss - it is pretty nuanced but surprisingly common.

Reply:
However for chat, if people are talking about sensitive subjects, you may not want a history of that data in the cloud for legal reasons.

Reply:
This is a demo app that anyone can join the public room, which may cause confusion among potential customers because they get messages from unknown people if the cloud sync is enabled.